src/Badges.sol

```diff
@@ -196,9 +196,23 @@ contract Badges is
     address _to,
     string calldata _uri,
     bytes calldata _signature
-  ) external virtual returns (uint256) {
+  ) public virtual returns (uint256) {
```

Comment:
Why would this be `public` when all the other badge minting functions here are external?